plugins/examples/security-authorization-engine/src/test/java/org/elasticsearch/example/CustomAuthorizationEngineTests.java

```diff
@@ -52,13 +47,15 @@ public void testGetAuthorizationInfo() {
 
     public void testAuthorizeRunAs() {
         final String action = "cluster:monitor/foo";
-        final TransportRequest request = new TransportRequest() {};
+        final TransportRequest request = new TransportRequest() {
+        };
         CustomAuthorizationEngine engine = new CustomAuthorizationEngine();
         // unauthorized
         {
-            Authentication authentication = Authentication
-                .newRealmAuthentication(new User("bar", "not_superuser"), new RealmRef("test", "test", "node"))
-                .runAs(new User("joe", "custom_superuser"), new RealmRef("test", "test", "node"));
+            Authentication authentication = Authentication.newRealmAuthentication(
+                new User("bar", "not_superuser"),
+                new RealmRef("test", "test", "node")
+            ).runAs(new User("joe", "custom_superuser"), new RealmRef("test", "test", "node"));
```

Comment:
No real change here other than format. This module has not been properly formatted for a while. There are a few more places in the PR like this.